app/views/govuk_publishing_components/components/docs/lead_paragraph.yml

```diff
@@ -20,3 +20,7 @@ examples:
       inverse: true
     context:
       dark_background: true
+  custom_margin_bottom:
+    data:
```

Comment:
Maybe add a description here explaining what can be passed and what it does? e.g. 6 adds 30px, etc.